packages/apps/plugins/plugin-space/src/components/SpacePresence.tsx

```diff
@@ -44,6 +47,13 @@ export const SpacePresence = ({ object, spaceKey }: { object: TypedObject; space
   const space = spaceKey ? client.spaces.get(spaceKey) : getSpaceForObject(object);
   const spaceMembers = useMembers(space?.key);
 
+  const [moment, setMoment] = useState(Date.now());
+
+  useEffect(() => {
```

Comment:
maybe just add a short note on why the intervals are needed. It makes a lot of sense (we need to update the UI w/o the data necessarily changing) but at first I was confused and I suspect I would be again when reading this code without the context of the issue